data/presets/pipeline/substation.json

```diff
@@ -0,0 +1,22 @@
+{
+    "icon": "temaki-powered_pump",
+    "fields": [
+        "name",
+        "substation",
```

Comment:
This field has strings for options which are specific for electrical substations. We should probably add a dedicated `substation/pipeline` field for this with the [corresponding values](https://wiki.openstreetmap.org/wiki/Key:substation#Pipeline_dedicated_substations).

Reply:
Thanks for catching this!